docs/content/docs/dev/datastream/operators/overview.md

```diff
@@ -755,5 +755,43 @@ someStream.filter(...).slotSharingGroup("name")
 ```python
 some_stream.filter(...).slot_sharing_group("name")
 ```
+
```

Comment:
Hi @wenlong88  If convenient could you also add the following two lines to fix the previous incomplete labels? Or  it could be put into a separate hotfix commit in this PR. 
```
{{< /tab >}}
{{< /tabs>}}
```